src/EpiRoute.php

```diff
@@ -151,8 +151,10 @@ public function run($route = false, $httpMethod = null)
 
     if (empty($routeDef))
       error_log('callback does not exist for path:' . $route . ' method: ' . $httpMethod);
-    
-    $response = call_user_func_array($routeDef['callback'], $routeDef['args']);
+  
+    $handler = call_user_func([$routeDef['callback'][0], 'getInstance']); // i() is not a good name
```

Comment:
for this one and in order to keep backwards compatibility
if we where doing something 
```php
if (method_exists($routeDef['callback'][0], 'getInstance')) {
    $handler = call_user_func([$routeDef['callback'][0], 'getInstance']);
    $response = call_user_func_array([$handler, $routeDef['callback'][1]], $routeDef['args']);
}
else {
     $response = call_user_func_array($routeDef['callback'], $routeDef['args']);
}
```

Reply:
it's in place now!